tests/asserts/final/publication/internal.py

```diff
@@ -191,3 +193,17 @@ def does_not_exist(workspace, publ_type, name, ):
     with app.app_context():
         pub_info = layman_util.get_publication_info(workspace, publ_type, name)
     assert not pub_info, pub_info
+
+
+def nodata_preserved_in_normalized_raster(workspace, publ_type, name):
+    with app.app_context():
+        publ_info = layman_util.get_publication_info(workspace, publ_type, name, {'keys': ['file']})
+    file_type = publ_info['file']['file_type']
+    if file_type == settings.FILE_TYPE_RASTER:
+        input_nodata_values = gdal.get_nodata_values(publ_info['_file']['gdal_path'])
+        input_nodata_value = input_nodata_values[0]
+        assert all(v == input_nodata_value for v in input_nodata_values)
+        normalized_nodata_values = gdal.get_nodata_values(publ_info['_file']['normalized_file']['path'])
+        normalized_nodata_value = normalized_nodata_values[0]
```

Comment:
I suggest to rename `normalized_nodata_value` to `normalized_first_band_nodata_value`